freeze.py

```diff
@@ -210,6 +214,27 @@ def find_files(directory, patterns):
     if os.path.exists(resvg_path):
         external_so_files.append((resvg_path, os.path.basename(resvg_path)))
 
+    # Add QtWebEngineProcess (if found)
+    web_process_path = "/usr/lib/x86_64-linux-gnu/qt5/libexec/QtWebEngineProcess"
+    external_so_files.append((web_process_path, os.path.basename(web_process_path)))
+
+    # Add QtWebEngineProcess Resources & Local
+    qt5_path = "/usr/share/qt5/"
```

Comment:
This one's probably fine as is, seems like it's the same place in every distro.

..Why do we need to package the WebEngine translation files, though? It's displaying exactly zero content, there's nothing to translate! Or does it just not run right if they're not there?

Reply:
I was unable to get this running without these files packaged up. However, I was testing many different things at all the same time, so it's possible I made a mistake. Feel free to verify.